docs/gotchas.md

```diff
@@ -40,3 +40,8 @@ class DistributorConfig:
 class Distributor(Component):
     def __init__(self, config: DistributorConfig = DistributorConfig()):
 ```
+
+## Installing from source
+- In order to edit the MAVA source code, you must ensure that you have cloned the repo and installed from source using `pip install -e .`
+- A common mistake is to `run pip install id-mava`, which installs MAVA from pypi, resulting in a duplicate version of MAVA being stored as a dependancy.
+- If you encounter `ModuleNotFoundError: No module named 'mava'` or Mava is executed by the example systems via site-packages (mava is stored as a package), a likely fix is to add the home directroy of the cloned Mava source code to `PYTHONPATH` with `export PYTHONPATH=<mava_homedir>`. On a linux-based, this command can be added to `.bashrc`, to avoid having to export the path every time a new terminal is opened.
```

Comment:
typo: "home DIRECTORY"

Reply:
typo: "on a linus-based SYSTEM"

Reply:
Think you can remove "Mava is executed by the example systems via site-packages (mava is stored as a package)"  as this is not fixed by PYTHONPATH but the issue is duplicate installation (covered by the bullet point above). Also, if you mention site-packages, you would need to mention venv.

Reply:
Perhaps you can move it to the bullet point above

Reply:
@omayma typo: "home DIRECTORY" still there

Reply:
Done!